specifyweb/frontend/js_src/lib/components/QueryBuilder/Wrapped.tsx

```diff
@@ -577,7 +577,8 @@ function Wrapped({
               <QueryResultsWrapper
                 createRecordSet={
                   !isReadOnly &&
-                  hasPermission('/querybuilder/query', 'create_recordset') ? (
+                  hasPermission('/querybuilder/query', 'create_recordset') &&
+                  !queryResource.get('selectDistinct') ? (
```

Comment:
can you use `query.selectDistinct` instead?

`queryResource.get('selectDistinct')` is not reactive (might not update the component on change)

Reply:
I tried with query.selectDistinct but then the issue is that when creating a query, select distinct, run the query, the createRecordSet is not displayed; but, if check the distinct box, even without running the query again, the button would appear as the state is changing. Hence my use of queryResource.get